### ISSUE TITLE
Contacts from Phone Storage (not visible by other apps) Are not displaying

### ISSUE THREAD

**Issue by storyjesse** [SimpleMobileTools/Simple-Dialer#657]
As the title says, contacts saved to Phone Storage (not visible by other apps) are not shown in the dialler contacts or call history.
I have the filter check box selected to display these contacts and can even create one through the dialler app saving it to Phone Storage (not visible by other apps) but it is not displayed. The created contact IS displayed in Simple Contacts Pro SE and the check box to "show private contacts to Simple Dialler... etc." Is selected in Simple Contacts Pro SE settings.

Simple Dialler is version 5.18.0 (FDroid)
Simple Contacts Pro SE is version 6.22.4.2 (FDroid)
I have tried installing Simple Contacts Pro version 6.22.4 (FDroid) but it makes no difference. It appears that Simple Dialler contains Simple Contacts Pro and installing the later has no effect.

I wonder if this is related to issue #463

**Comment by tibbi** [comment]
can you reproduce it reliably by creating a new contact?

**Comment by blakegearin** [comment]
> can you reproduce it reliably by creating a new contact?

Yes. Creating a new contact through Dialer successfully adds the contact to phone storage and doesn't show up in Dialer afterwards.
